Update improving-your-stores-performance.md (#713)
vtex-apps/io-documentation: docs/en/Recipes/store-management/improving-your-stores-performance.md

# Improving your store's performance

A common worry among e-commerce stores is site performance.

Such worries are more than understandable. Performance directly impacts the sales conversion rate, user session time, bounce rate, and even the number of visited pages and site findability. Consequently, to guarantee your brand's online success, you must look after your site's performance and how fast its content is displayed to end-users.

> Thus, we present, in the following, some practices we recommend and that you can instantly implement in your store to ensure a better end-user experience.

Reviewer: ```suggestion
This article contains recommendations you can implement in your store to improve site performance and, as a consequence, shopper experience.
```

Instantly só se aplica se for um toggle. Se eu tiver que pensar por mais que um segundo, acho melhor não usar pra descrever as recomendações.

> :information_source: *Once adopted, such practices can lead to an SEO improvement of up to 80% and over 50% in page loading time. This data can be identified in the main website performance analysis tools, such as Google Lighthouse and Google Analytics Load Time.*

Reviewer: ```suggestion
:information_source: *Once adopted, such practices can lead to an improvement of up to 80% in SEO and over 50% in page loading time. This data can be identified in the main website performance analysis tools, such as [Google Lighthouse](https://developers.google.com/web/tools/lighthouse) and [Google Analytics](https://support.google.com/analytics/answer/1205784?hl=en).*
```

Reviewer: Inverti a ordem no SEO pra melhorar o paralelismo, sempre bom também linkar para páginas sobre ferramentas externas.

# Best practices

> Between the multiple actions that you can take to boost your store's website performance, you can instantly enable some of them through the Administrative Panel, under *Store Setup > CMS > Store > Advanced*. 

Reviewer: ```suggestion
Some actions that can boost your store's website performance can be enabled in your store's Admin Panel, under *Store Setup > CMS > Store > Advanced*. 
```
